### PR TITLE
KOGITO-1476: Update README with instructions on master vs. stable

### PR DESCRIPTION
**Pull request by evacchi**

### CODE REVIEW

**Comment by radtriste** [comment]
Do we have a PR template where we can put the warning about PR to `master` branch ?

**Comment by evacchi** [comment]
nice suggestion!

**Comment by evacchi** [comment]
of course, this will only work when it lands on `stable`...

**Comment by radtriste** [comment]
at least we have a warning if PR against stable

**Review by ricardozanini — APPROVED** [review]

**Review by cristianonicolai — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,7 +4,19 @@ This module contains a number of examples that you can take a look at and try ou
...
+- Default branch is `stable`, pointing to the latest released version. 
+- **[You can also check all versions by looking at releases.](https://github.com/kiegroup/kogito-examples/releases/latest)**
+
+
+## Contribution
+
+Everyone is encouraged to contribute to this quickstarts by
+
+* trying it out and providing feedback and ideas for improvement
+* create new quickstarts -- **in this case, make sure your PR is against the `master` branch!**
```
Comment (radtriste) [inline README.md]: `create new examples`

File: README.md [hunk 2]
```diff
@@ -4,7 +4,19 @@ This module contains a number of examples that you can take a look at and try ou
...
 Since Kogito aims at supporting both Quarkus and SpringBoot each example usually provides both type of projects.
 
-**To try it out locally on latest stable version download the released repository content [here](https://github.com/kiegroup/kogito-examples/releases/latest)**
+- Default branch is `stable`, pointing to the latest released version. 
+- **[You can also check all versions by looking at releases.](https://github.com/kiegroup/kogito-examples/releases/latest)**
+
+
+## Contribution
+
+Everyone is encouraged to contribute to this quickstarts by
```
Comment (radtriste) [inline README.md]: quickstarts => examples

**Review by radtriste — APPROVED** [review]
minor comments

**Pull request merged**